pages/community/precice-workshop-2024.md

```diff
@@ -259,6 +259,56 @@ The cost of lunch, as well as coffee and snacks is included in the registration
 - 15:00-15:30: ☕️ Coffee break
 - 15:30-17:30: Hands-on user support
 
+## Program details
+
+### User introductions
+
+The official welcome to the workshop on Wednesday 25 will be followed by a round of introductions, so that everyone knows who to talk to during the workshop.
```

Comment:
```suggestion
The official welcome to the workshop on Wednesday, September 25. It will be followed by a round of introductions, so that everyone knows who to talk to during the workshop.
```